### DISCUSSION TITLE
Pool Rewards not happening since 1.2.5

### DISCUSSION THREAD

**Discussion by mdio33** [Chia-Network/chia-blockchain#8283] [Q&A]
Hello.
I've got a few plots with almost no change to win, but after the pool farming I've started to recieve small amounts every day, 2 or 3 per day.
Now, after upgrading to 1.2.5 those rewards stopped..

Anyone know why  that happen?

thank you.

![chia](https://user-images.githubusercontent.com/44021265/131410775-2a6d74c1-9ef7-4328-b709-68d371213d92.jpg)

**Comment by loppefaaret** [comment]
First, confirm that your wallet is in sync - then try and check up on changes in payout structures from your pool

**Reply by mdio33** [reply]
TY for your answer.
The wallet is synchronized .

![Captura de ecrã 2021-08-31 201146](https://user-images.githubusercontent.com/44021265/131562642-92c74645-da4c-454f-88c8-9f7e8b210bd6.jpg)

Sorry for the newbie questions, but how can I check payout structures of my pools?

What i can see is that the Payout Adress is not my wallet address, is that ok?

**Reply by loppefaaret** [reply]
your wallet has many many possible receive addresses - whenever it has received a transaction using one, it will use the next, constantly generating new addresses

**Comment by loppefaaret** [comment]
check how your pool is relying information - it can be through a discord chat, twitter, on their pool page...

**Reply by mdio33** [reply]
It must be something else, I've got 79 plots distributed by 3 different pools and i'm not getting any rewards.

Is there any way to remove the pools i have, then re-add it and after assign the plots to them?

**Reply by loppefaaret** [reply]
you could leave the pool by switching your plotNFT to self local farming, and back onto the pool after i guess - but i doubt that makes any difference - check the pools if you are getting your partials registered on them ? do the pools report payouts ?

**Comment by BrandtH22** [comment]
@mdio33 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
